.github/workflows/documentation.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         with:
           version: '1'
       - name: Install dependencies
-        run: julia --project=docs/ -e 'using Pkg; Pkg.develop(path="./NDTensors"); Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+        run: julia --project=docs/ -e 'using Pkg; Pkg.develop(path="./NDTensors"); Pkg.develop(PackageSpec(path=pwd()));'
```

Comment:
Could you explain this change?

Reply:
No this must have been an accidental change from when I was looking into a problem ryan had